pkgs/by-name/nf/nfd/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  boost, # probably needs to match the one from ndn-cxx
+  fetchFromGitHub,
+  libpcap,
+  ndn-cxx,
+  openssl,
+  dpkg,
+  pkg-config,
+  sphinx,
+  systemd,
+  wafHook,
+  websocketpp,
+  withSystemd ? lib.meta.availableOn stdenv.hostPlatform systemd,
+  withWebSocket ? true,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nfd";
+  version = "24.07";
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = lib.toUpper pname;
```

```suggestion
    repo = "NFD";
```
we don't want to do to many assumptions about pname

---

Fixed